Fastfile

```diff
@@ -78,7 +78,7 @@ def _setup
   ENV['SCAN_DEVICE'] ||= 'iPhone 6 (9.3)'
   xcode_select(ENV['TAB_XCODE_PATH']) if is_ci && !ENV['TAB_XCODE_PATH'].nil?
 
-  unless ENV['TAB_UI_TEST_SCHEME'].nil?
+  unless ENV['TAB_UI_TEST_SCHEME'].nil? # rubocop:disable Style/GuardClause
```

Comment:
If you disabled this one for your knowledge then it wants us to do:
```ruby
return unless ENV['TAB_UI_TEST_SCHEME'].nil?
```
rather than it being in a closure. This is a good disable to leave in though because otherwise it would exit out and we don't want that as the scheme might genuinely be empty